recipes/mercurial-app/meta.yaml

```
@@ -0,0 +1,43 @@
+# metapackage for a Mercurial "conda application" providing
+# Mercurial with few extensions (hg-evolve and hg-git)
+
+# Note: hg-git is not yet available on conda-forge
+
+# inspired by
+# https://github.com/conda-forge/mpi-feedstock
+# https://github.com/conda-forge/spyder-feedstock
+
+package:
+  name: mercurial-app
+  version: 1.0
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - python <3
+    - mercurial
+    - hg-evolve
+    - dulwich
+    - ipaddress  # [py==27]
+
+test:
+  commands:
+    - hg --version
+
+app:
+  entry: hg
+  icon: logo.png
+  summary: Source control management tool
+  own_environment: True
+
+about:
+  home: https://github.com/conda-forge/mercurial-app-feedstock
+  license: GPL2
+  license_file: {{ RECIPE_DIR if RECIPE_DIR is defined else '' }}/LICENSE.txt
```

There is no need for the `RECIPE_DIR` part as conda automatically looks there if it isn't it the build directory
```suggestion
  license_file: LICENSE.txt
```